model/ride.js

```diff
@@ -172,14 +172,19 @@ const listPendingRidesForDriver = async (user, db) => {
   return db
     .any(
       `
-    SELECT a.driver, a.date, a.time, a.origin, a.destination, a.car FROM advertisedCarRide a
-    LEFT OUTER JOIN bid b ON a.driver = b.driver
-      AND a.date = b.date
-      AND a.time = b.time
-      AND a.origin = b.origin
-      AND a.destination = b.destination
-    WHERE a.driver = $1 AND (b.bidStatus = 'pending' OR b.bidStatus IS NULL)
-    GROUP BY a.driver, a.date, a.time, a.origin, a.destination;
+      SELECT DISTINCT a.driver, a.date, a.time, a.origin, a.destination, a.car 
+      FROM advertisedCarRide a, bid b
```

Comment:
In this case, do we still need the table bid b?

Reply:
Thanks , nice spot